libvast/src/system/explorer.cpp

```diff
@@ -134,23 +135,23 @@ explorer(caf::stateful_actor<explorer_state>* self, caf::actor node,
         VAST_DEBUG(self, "could not find timestamp field in", layout);
         return;
       }
-      std::optional<table_slice::column_view> by_column;
+      std::optional<table_slice_column> by_column;
```

Comment:
This exemplifies my issue with the "nullability" of `table_slice_column`. With the way it is defined right now, the `optional` is redundant.